screens/Home.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button, Alert, Image, TextInput, StackNavigator} from 'react-native';
+import { Font } from 'expo';
+
+export class HomeScreen extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./../assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./../assets/fonts/SourceSansPro-Regular.ttf'),
+    });
+    this.setState({ fontLoaded: true});
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={{backgroundColor: '#C2515B', height: '80%'}}>
+          <View style={{flexDirection: 'row', paddingTop: 30}}>
+            <View style={styles.shadowPong}>
+              <View style={styles.pongContainer}>
+              {
+                this.state.fontLoaded ? (
+                  <Text style={styles.pongText}>Pong</Text>
+                ) : null
+              }
+                <View style={styles.matesContainer}>
+                  <Text style={styles.matesText}>Mates</Text>
+                </View>
+              </View>
+            </View>
+          </View>
+          <View style={{alignItems: 'center', paddingTop: 55}}>
+            <View style={styles.shadowHome}>
+              <View style={styles.homeRectangle}>
+                <View style={{borderRadius: 20}}>
+                  <View style={{backgroundColor: '#FFC928', width: 200, height: 200, borderRadius: 100}}>
+                    <Image style={{width: 200, height: 200}} source={require('./../assets/image.png')}/>
+                    <Text style={styles.selfieText}>Take a selfie to share with your partner!</Text>
+                  </View>
+                </View>
+                <View style={{paddingTop: 40}}>
+                  <View style={styles.shadowName}>
+                    <View style={styles.nameContainer}>
+                      <TextInput
+                        style={styles.nameInput}
+                        placeholder="Your Name"
+                        onChangeText={(text) => this.setState({text})}
+                      />
+                    </View>
+                  </View>
+                </View>
+              </View>
+            </View>
+          </View>
+          <View style={{paddingTop: 20}}>
+            <Button
+              onPress={() =>
+                navigate('SelectionScreen')
```

Comment:
This can be placed on its own line: `onPress={() => navigate('SelectionScreen') }`